source/administration/command-line-tools.rst

```diff
@@ -1611,8 +1612,24 @@ mattermost webhook create-outgoing
           --icon [iconURL]                        Icon URL
           --url stringArray [REQUIRED]            Callback URLs 
           --content-type string                   Content-type
-	  --h, --help				  Help for create-outgoing
-	  
+    --h, --help         Help for create-outgoing
```

Comment:
Here you are channging the identation, is expected to be at the same level than the previous one in both columns

Reply:
Fixed - I must have accidentally hit delete.